src/run.ts

```diff
@@ -118,12 +165,24 @@ function findHelm(rootFolder: string): string {
 
 async function run() {
     let version = core.getInput('version', { 'required': true });
-    if (version.toLocaleLowerCase() === 'latest') {
-        version = await getStableHelmVersion();
-    } else if (!version.toLocaleLowerCase().startsWith('v')) {
-        version = 'v' + version;
+
+    if (process.env['NEW_VERSION_LOGIC'] == 'true') {
```

Comment:
What I had in mind was following the new versioning model by default and having an env variable to unblock if any issues are faced. Something like:

```typescript
if (process.env['HELM_INSTALLER_LEGACY_VERSIONING'] == 'true') {
  // old versioning logic
}
else {
  // new versioning logic
}
```

Reply:
Ohh, got it. I'll do that.